$OpenBSD: patch-config_mak_uname,v 1.6 2018/07/06 20:15:42 kn Exp $

OpenBSD wants DIR_HAS_BSD_GROUP_SEMANTICS, just like FreeBSD

Index: config.mak.uname
--- config.mak.uname.orig
+++ config.mak.uname
@@ -215,6 +215,7 @@ endif
 ifeq ($(uname_S),OpenBSD)
 	NO_STRCASESTR = YesPlease
 	NO_MEMMEM = YesPlease
+	DIR_HAS_BSD_GROUP_SEMANTICS = YesPlease
 	USE_ST_TIMESPEC = YesPlease
 	NEEDS_LIBICONV = YesPlease
 	BASIC_CFLAGS += -I/usr/local/include
